docs/install/configurations/environment-variables.mdx

```diff
@@ -33,7 +31,7 @@ To configure activepieces, you will need to set some environment variables, Ther
 | `AP_PIECES_SYNC_MODE`              | `AP_PIECES_SYNC_MODE`: `NONE` for no metadata syncing / 'OFFICIAL_AUTO' for automatic syncing for pieces metadata from cloud                                                                                         | `OFFICIAL_AUTO`                                                                |
 | `AP_POSTGRES_DATABASE`             | ❗️ The name of the PostgreSQL database                                                                                                                                                                              | `None`                                                                         |                                                                        |
 | `AP_POSTGRES_HOST`                 | ❗️ The hostname or IP address of the PostgreSQL server                                                                                                                                                              | `None`                                                                         |                                                                        |
-| `AP_POSTGRES_PASSWORD`             | ❗️ The password for the PostgreSQL user                                                                                                                                                                             | `None`                                                                         |                                                                        |
+| `AP_POSTGRES_PASSWORD`             | ❗️ The password for the PostgreSQL user is a 32-character hexadecimal key. You can generate one using Open SSL.                                                                                                                                                                              | `None`                                                                         |                                                                        |
```

Comment:
The postgres password can be anything but you can offer to generate one using `openssl rand -hex 32`

You can add this comment